cumulusci/tasks/metadata_etl/help_text.py

```diff
@@ -0,0 +1,87 @@
+from collections import defaultdict
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.tasks.metadata_etl import MetadataSingleEntityTransformTask
+from cumulusci.utils.xml.metadata_tree import MetadataElement
+from cumulusci.core.utils import process_list_arg, process_bool_arg
+
+
+class SetFieldHelpText(MetadataSingleEntityTransformTask):
+    entity = "CustomObject"
+    task_options = {
+        "fields": {
+            "description": "List of object fields to affect, in Object__c.Field__c form.",
+            "required": True,
+        },
+        "overwrite": {
+            "description": "List of object fields to affect, in Object__c.Field__c form."
+        },
+        **MetadataSingleEntityTransformTask.task_options,
+    }
+
+    def _init_options(self, kwargs):
+        self.task_config.options["api_names"] = "dummy"
+        super()._init_options(kwargs)
+
+        self.options["overwrite"] = process_bool_arg(
+            self.options.get("overwrite", False)
+        )
+
+        try:
+            float(self.api_version)
+        except ValueError:
+            raise TaskOptionsError(f"Invalid API version {self.api_version}")
+
+        if type(self.options["fields"]) != list or len(self.options["fields"]) == 0:
```

Comment:
```suggestion
        if not isinstance(self.options.get("fields"), list) or len(self.options["fields"]) == 0:
```

Changed to use isinstance which is more idiomatic in Python (it also matches subclasses, though that doesn't matter here) and to use .get() in case the user forgot to provide the fields option (validation for "required" options doesn't take place until after _init_options)